tyk-control-plane/values.yaml

```diff
@@ -228,10 +228,11 @@ global:
     # It is used to set TYK_MDCB_SECURITY_PRIVATECERTIFICATEENCODINGSECRET
     privateCertificateEncodingSecret: ""
 
-  # hashKeys specifies that if your Tyk Gateway is using hashed keys, set this value to true, so it matches.
-  # The Dashboard will now operate in a mode that is compatible with key hashing.
-  # It will enable hash keys for Tyk Gateway
-  # It is used to set TYK_GW_HASHKEYS
+  # hashKeys specifies that if your Tyk Gateway is using hashed keys, set this value to true so it matches.
+  # The Dashboard and MDCB will now operate in a mode that is compatible with key hashing.
+  # This takes precedence over the dashboard value i.e `dashboard.hashKeys`
+  # It is used to set TYK_DB_HASHKEYS and TYK_MDCB_HASHKEYS
+  # It will enable hashkeys in both mdcb and dashboard.
```

Comment:
`.global.hashKeys ` also applies to Tyk Gateway as well. So, it also configures `TYK_GW_HASHKEYS`

Reply:
Updated